suite-native/atoms/src/Skeleton/ListItemSkeleton.tsx

```diff
@@ -16,7 +16,7 @@ const MAIN_ITEM_HEIGHT = 48;
 const SUBITEM_HEIGHT = 20;
 
 const SCREEN_WIDTH = Dimensions.get('window').width;
-const MAIN_ITEM_WIDTH = SCREEN_WIDTH * 0.56;
+const MAIN_ITEM_WIDTH = SCREEN_WIDTH * 0.6;
```

Comment:
We should make this dynamic by container size so we don't need to change this with every slight padding change :D

Reply:
It shouldn't happen that we change this often. The fix yesterday was redundant so I just returned it and it should stay as it is. 

AFAIK dynamic size is not possible due to skia. 
@Nodonisko